cloudinit/config/cc_refresh_rmc_and_interface.py

```diff
@@ -25,23 +25,29 @@
 - Refreshing RMC
 - Disabling NetworkManager from handling IPv6 interface, as IPv6 interface
   is used for communication between RMC daemon and PowerVM hypervisor.
-
-**Internal name:** ``cc_refresh_rmc_and_interface``
-
-**Module frequency:** always
-
-**Supported distros:** RHEL
-
 """
 
 import errno
 
 from cloudinit import log as logging
 from cloudinit import netinfo, subp, util
+from cloudinit.config.schema import MetaSchema
+from cloudinit.distros import ALL_DISTROS
 from cloudinit.settings import PER_ALWAYS
 
 frequency = PER_ALWAYS
 
+# This module is undocumented
+meta: MetaSchema = {
+    "id": "cc_refresh_rmc_and_interface",
+    "name": "Refresh IPv6 Interface and RMC",
+    "title": "Ensure Network Manager is not managing IPv6 interface",
+    "description": __doc__,
+    "distros": [ALL_DISTROS],
+    "frequency": PER_ALWAYS,
```

Comment:
Might want to still have the local 'frequency' variable in case someone changes that value and doesn't notice this one in the future.
```suggestion
    "frequency": frequency,
```